### PR TITLE
Update SBT and plugins

### PR DESCRIPTION
**Pull request by guersam**
Update to SBT 0.13.12 as well as the plugins.

### CODE REVIEW

File: project/build.properties [hunk 1]
```diff
@@ -1 +1 @@
-sbt.version=0.13.5
+sbt.version=0.13.13-RC3
```
Comment (satabin) [inline project/build.properties]: Is it really necessary to upgrade to an RC version?

**Comment by satabin** [comment]
As a global remark, I noticed that formatting changes a lot, is it due to the new scalariform version? I am reluctant to merge this change if it reformat everything, wehereas only the build file must have been changed.

File: project/build.properties (hunk 1 above)
Comment (guersam) [inline project/build.properties]: Not actually, I just wanted to avoid as many future breaking changes as possible. 0.13.12 is totally fine to me.

**Comment by guersam** [comment]
Ok, I'll try SBT 0.13.12 with the same sbt-scalariform version as before.

**Comment by satabin** [comment]
There probably is a setting in the new scalariform version to have the same formatting as before. I will investigate.

**Comment by guersam** [comment]
I reverted scalariform for now, because to keep the format as before we need `FirstParameterOnNewline` preference which is not included in the latest version yet.

**Comment by satabin** [comment]
LGTM, thanks for the update

**Pull request merged**